internal/ui/static/js/app.js

```diff
@@ -782,7 +781,7 @@ function handleMediaControl(button) {
     enclosures.forEach((enclosure) => {
         switch (action) {
         case "seek":
-            enclosure.currentTime = enclosure.currentTime + value > 0 ? enclosure.currentTime + value : 0;
+            enclosure.currentTime = Math.min(enclosure.currentTime + value, 0);
```

Comment:
Shouldn't this be `Math.max`?

Reply:
Err, it should indeed. I sent #3086 to fix this, thanks!